packages/network/src/logic/trackerSummaryUtils.ts

```diff
@@ -66,6 +67,19 @@ export function addRttsToNodeConnections(
     }
 }
 
+export function getNodesWithLocationData(nodes: ReadonlyArray<string>, locations: Readonly<{[key: string]: Location}>): NodesWithLocations {
```

Comment:
Maybe a test for this? E.g. in "trackerSummaryUtils.test.ts" or (indirectly) thru integraiton test  "tracker-endpoints.test.ts". The latter file contains a test for the usual case, but it would be good to get the "location does not exist" case under test as well.